### ISSUE TITLE
Undo hotkey Alt+Backspace makes a beep sound

### ISSUE THREAD

**Issue by ygoe** [notepad-plus-plus/notepad-plus-plus#898]
Windows traditionally uses both hotkeys for undo, Alt+Backspace and Ctrl+Z. (There are more alternates often neglected by developers, like Ctrl+C/Ctrl+Insert, Ctrl+X/Shift+Delete, Ctrl+V/Shift+Insert.)

While both undo hotkeys work here, fortunately (unlike Ctrl+W vs. Ctrl+F4, see #879), Alt+Backspace plays the default sound when pressed. This should not happen.

**Comment by NN---** [comment]
Alt+Backspace is working as Undo for a long time.
Check your settings, maybe uninstall and clean the profile.

**Comment by dg9ngf** [comment]
NN, please turn on your speakers and enable the default sound profile, then you hear what I mean.

**Comment by NN---** [comment]
@dg9ngf Nice :)
I always work with "No sound" profile

**Comment by MetaChuh** [comment]
we are closing this issue due to its age and multiple newer notepad++ versions since it was filed.

please update your notepad++ to the latest version [<kbd>Notepad++ 7.6.6</kbd>](https://notepad-plus-plus.org/download/v7.6.6.html).
if your issue persists, and it is still of relevance to you, please feel free to file a new issue.

thank you for your help and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
```
[<kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd>](https://notepad-plus-plus.org/community/)

**Comment by ygoe** [comment]
Confirmed fixed in 7.6.6.

**Comment by martinlebrun-vertexinc** [comment]
The beep on <Alt><Backspace> still happens in v7.8.7

**Comment by sasumner** [comment]
A **_LOT_** of keycombos that include Alt make some sort of sound, not just Backspace.
I'd try my hand at fixing them if I knew more about how to do it. :-(

**Comment by donho** [comment]
@sasumner I thought you love ringing the bell ;)

**Comment by sasumner** [comment]
Ringing the bell is no problem for me.
Stopping it from ringing when it should not -- that's my problem. :-)

**Comment by ygoe** [comment]
Oh, NPP now also makes some noise when I press F3 to find the next occurrence and there is none. Or was it when I Ctrl+F find something that isn't there? At least the search function is not always silent. Maybe this is related?

**Comment by sasumner** [comment]
>press F3 to find the next occurrence and there is none. 

>Or was it when I Ctrl+F find something that isn't there?

Yes a noise is made for either of these situations (which are really equivalent).

>Maybe this is related?

No, because there is no Alt key involved in either of those.
The Alt key is important to this issue.

**Comment by Bison700** [comment]
It still does it today. Very annoying. I wanted to map SCI_LINEDELETE to Alt+Backspace, but it always plays the default Windows sound.

**Comment by Mohsyn** [comment]
I use Alt X to delete line [SCI_LINEDELETE] and there is this extremely annoying windows default beep sound every time a line is deleted. This is a perennial issue it seems.

**Comment by alankilborn** [comment]
Current status:  Notepad++ 8.5.8 still has this problem (for Alt+Backspace or "Alt+letter that isn't a menu activator e.g. Alt+f for _File_, Alt+e for _Edit_, etc.)

I found some coding references on stopping Alt keycombos from making sound, but nothing definitively points to a solution:

* https://stackoverflow.com/questions/74998972/windows-programming-and-key-events-alt-key-sound-effect
* https://stackoverflow.com/questions/75295012/remove-windows-error-beep-sound-when-pressing-alt-key-combinations
